### PR TITLE
Update ja_jp.lang

### PR DESCRIPTION
**Pull request by GukkyGukz**

### CODE REVIEW

**Comment by ChAoSUnItY** [comment]
> container.eki.ticket_vendor.state.permission=削除 <-- Insufficient permissions.
> container.eki.ticket_vendor.state.withdraw=発券 <-- Successfully Withdrawed!  
Wrong meaning?

**Comment by GukkyGukz** [comment]
Fixed

File: src/main/resources/assets/eki/lang/ja_jp.lang [hunk 1]
```diff
@@ -137,15 +137,15 @@ tile.eki.anchor.warning.tooltip=バインドされた場合はこのブロック
 
 
 //GUIS
-container.eki.ticket_vendor.state.success=発券しました！
+container.eki.ticket_vendor.state.success=発券しました！残高：%d
```
Comment (ChAoSUnItY) [inline src/main/resources/assets/eki/lang/ja_jp.lang:140]: there's no %d (integer) parameter provided in this message, will cause format error.

Also, withdraw means get the money out of the vendor, money is saved when player successfully tickets a ticket from vendor with specific amount of money. Maybe first time button meaning is right.

**Review by ChAoSUnItY — CHANGES_REQUESTED** [review]
review done, still some error (1 fatal error & maybe 2 meaning confused).

File: src/main/resources/assets/eki/lang/ja_jp.lang (hunk 1 above)
Comment (ChAoSUnItY) [reply src/main/resources/assets/eki/lang/ja_jp.lang:140]: sorry, my fault, it's not a fatal format error, but its value means how much money you request to be apply to a ticket. Maybe translate from "Value"?

**Comment by GukkyGukz** [comment]
”Ticket value” in japanese is usually translated to 残高, meaning how much value is left.

**Pull request merged**